### ISSUE TITLE
CSS minification breaks theming

### ISSUE THREAD

**Issue by KeithClinard** [knoxville-utilities-board/nrg-ui#75]
Given CSS in the form of:
```css
[data-theme="themename"] {
  --variable: value;
  .btn{
    --btn-variable: value;
  }
}
```

CSS minification produces the following structure:
```css
[data-theme="themename"] {
  --variable: value;
}
.btn{
  --btn-variable: value;
}
```

This effectively breaks theming by "unwrapping" nested styles from the main `data-theme` attribute.

**Comment by KeithClinard** [comment]
A temporary fix has been put in place by removing CSS minification. PR #74

**Comment by KeithClinard** [comment]
This was resolved by #245.
